urx/robot.py

```diff
@@ -214,20 +218,26 @@ def new_csys_from_xpy(self):
         Return new coordinate system from three points: X, Origin, Y
         based on math3d: Transform.new_from_xyp
         """
+        # Set coord. sys. to 0
+        self.csys = m3d.Transform()
```

Comment:
I agree, how should we do it?

a) Restore csys inside the method, and don't return the result
b) Restore csys inside the method, and return the result
c) Return result as now, and specify in docstring that user must restore csys afterwards